scripts/build-command-op-templates

```diff
@@ -108,7 +109,8 @@ def get_properties(p):
             for i in lists:
                 comp_exprs.append("echo \"{0}\"".format(i.text))
             for i in paths:
-                comp_exprs.append("/bin/cli-shell-api listActiveNodes {0} | sed -e \"s/'//g\" && echo".format(i.text))
+                path = re.sub(r'\s+', '/', i.text)
+                comp_exprs.append("ls /opt/vyatta/config/active/{0} 2>/dev/null".format(path))
```

Comment:
Please de-hardcode the path to `vyos.defaults.directories`

https://github.com/vyos/vyos-1x/blob/current/python/vyos/defaults.py#L38

Reply:
This script is only executed at build time and doesn't have `vyos.*` modules in its environment, so we'll need to either mess with Python path in the Makefile or import by absolute path. I don't think it's worth it in this case.

Reply:
It's not worth the trouble.